Avhw pdr 2

msglazer opened the pull request:
hardware PDR should be ready to add into the site

msglazer committed:
Create pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

msglazer committed:
Update pdr_avhw.md

philiplinden commented on docs/pdr_avhw.md:
```diff
@@ -0,0 +1,619 @@
+
+Lead: Matthew Glazer
+
+[TOC]
```
This table of contents is suuuper long. Maybe manually pick out a few links to the top level items and place it here instead.

msglazer replied:
is it better now?

philiplinden commented on docs/pdr_avhw.md:
```diff
@@ -0,0 +1,619 @@
+
+**Overview**: The Flight Hardware system for the nucleus mission provides all the power, computing and control elements required for mission success. There are seven primary PCBs and an assortment of secondary PCBs as part of this system.
+
+### Boards in work:
```
Can probably cut this boards in work section. It was relevant for the review but isn't really relevant on the site and will get stale quick.

msglazer replied:
removed

philiplinden commented on docs/pdr_avhw.md:
```diff
@@ -0,0 +1,619 @@
+##### Connectors
+There is one 40 pin Molex MilliGrid vertical receptacle, one 34 pin Molex MilliGrid vertical receptacle and six 28 pin Molex MilliGrid vertical receptacles providing the interfaces between the various avionics cards. There is also a Molex Microfit header and RJ45 connector for ground station interface and a 10 pin JST header for connection to the balloon board.
+
+###### Main Flight Computer Interface
```
If there's only 1 line of text after a heading, it might work better as a table?

msglazer replied:
i switched it to bold text, with the further sub headings being italics

philiplinden commented on docs/pdr_avhw.md:
```diff
@@ -0,0 +1,619 @@
+###### SDR
+An off the shelf SDR will be mounted to the communications card, it will communicate with the MFC through a USB 2.0 link. 
+
+###### RF Front End
```
comment out empty sections

msglazer replied:
done

philiplinden committed:
Add avhw to sidebar

philiplinden reviewed (changes requested):
Consider formatting things as just bold text or putting it in a table rather than headings 5 levels deep.

msglazer committed:
Update pdr_avhw.md

Removed boards in work
hid the RF front end section
fixed the MCU part number

msglazer committed:
testing different text style

msglazer committed:
trimming out TOC links

msglazer committed:
MORE TRIMMING

msglazer reviewed (commented):
i've realized there's a decent amount of info missing but let's get it live

msglazer committed:
maybe,,,,,finally.....

philiplinden commented on docs/pdr_avhw.md:
```diff
@@ -0,0 +1,672 @@
+
+Battery pack will be tab welded together for electrical connections, cells well be hotglued together for initial mechanical stability with additional framing to be added as necessary.
+
+Testing
```
Should this be a heading or removed?

msglazer committed:
Update pdr_avhw.md

msglazer commented:
@philiplinden I fixed that testing thing, it was missing its header info

in the battery pack area, changed manufacturing to "pack assembly"

Ill merge this in now
